backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -3,133 +3,107 @@
   }
 
   protected void useOutsideOrgUser() {
-    LoggerFactory.getLogger(BaseApiTest.class)
-        .info("Configuring auth service mock for outside org user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(OUTSIDE_USER_ORG_ROLES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring auth service mock for outside org user");
+    _userName = "intruder@pirate.com";
   }
 
   protected void useOrgAdmin() {
-    LoggerFactory.getLogger(BaseApiTest.class).info("Configuring auth service mock for org admin");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(ADMIN_ORG_ROLES);
-    _initService.initCurrentUser();
+    _userName = "admin@example.com";
+    LOG.info(
+        "Configuring auth service mock for org admin: {}",
+        _users.getByUsername(_userName).getAuthorization().getGrantedPermissions());
   }
 
   protected void useOutsideOrgAdmin() {
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(OUTSIDE_ADMIN_ORG_ROLES);
-    _initService.initCurrentUser();
+    _userName = "captain@pirate.com";
   }
 
   protected void useOrgEntryOnly() {
-    LoggerFactory.getLogger(BaseApiTest.class)
-        .info("Configuring auth service mock for org entry-only");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(ENTRY_ONLY_ORG_ROLES);
-    _initService.initCurrentUser();
+    _userName = "nobody@example.com";
   }
 
   protected void useSuperUser() {
-    LoggerFactory.getLogger(BaseApiTest.class).info("Configuring supplier mock for super user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.SITE_ADMIN_USER_ATTRIBUTES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring supplier mock for super user");
+    _userName = TestUserIdentities.SITE_ADMIN_USER;
   }
 
-  protected void setRoles(Set<OrganizationRole> roles) {
-    List<OrganizationRoleClaims> orgRoles;
-    if (roles != null && !roles.isEmpty()) {
-      orgRoles = Collections.singletonList(new OrganizationRoleClaims("DIS_ORG", roles));
-    } else {
-      orgRoles = Collections.emptyList();
-    }
-    when(_authService.findAllOrganizationRoles()).thenReturn(orgRoles);
+  protected void useBrokenUser() {
+    _userName = "castaway@pirate.com";
   }
 
   @BeforeEach
   public void setup() {
     truncateDb();
     _oktaRepo.reset();
+    TestUserIdentities.withStandardUser(
+        () -> {
+          _dataFactory.createValidOrg("DataLorg", "DAT_ORG");
+          _initService.initAll();
+        });
     useOrgUser();
-    _initService.initAll();
+    LOG.warn("Current configuration default user is {}", _users.getDefaultUser());
+    LOG.warn(
```

Comment:
feels too verbose to happen before every test -- is there any place better for this, given that it's gonna print the same thing for every test?

Reply:
Agreed—I think probably strip it out, now that the wiring is fixed. Or boot it down to `trace` in case we break it again in the future?

Reply:
trace is fine!